dataStorage/storage.js

```diff
@@ -67,17 +67,24 @@ async function loadFromCloudAsync(model, id) {
     try {
         // fetch data from service
         const response = await fetch(url);
-        // FIXME: [Wei] "response.json()" triggers error on Android, so I have to use "eval"
-        // responseJson = eval("(" + response._bodyText + ")")
-        responseString = await response.text();
-        responseJson = JSON.parse(responseString);
+
+        try {
+            // FIXME: [Wei] "response.json()" triggers error on Android, so I have to use "eval"
+            // responseJson = eval("(" + response._bodyText + ")")
+            responseString = await response.text();
+            responseJson = JSON.parse(responseString);
```

Comment:
can you please use this?

if (Platform.OS === 'ios') {
            responseJson = response.json();
        } else {
            const responseString = await response.text();
            responseJson = eval("(" + responseString + ")");
        }